packages/loopring_v3/contracts/aux/access/LoopringIOExchangeOwner.sol

```diff
@@ -114,10 +116,11 @@ contract LoopringIOExchangeOwner is SelectorBasedAccessManager, ERC1271, Drainab
             "INVALID_DATA"
         );
 
+        // Decode the blocks
```

Comment:
add a comment:

// Decode the blocks so `_beforeBlockSubmission` next line may modify the auxiliary data fields to reflect pre-approvals.